tools/Generator/CodeGenerator.fs

```diff
@@ -75,7 +75,7 @@ module CodeGenerator =
         w
 
     let generateCreateFunction (data: CreateData) (w: StringWriter) =
-        w.printfn "    static member val CreateFunc%s : (unit -> %s) = (fun () -> ViewBuilders.Create%s())" data.Name data.FullName data.Name
+        w.printfn "    static member val CreateFunc%s : (unit -> %s) = (fun () -> ViewBuilders.Create%s()) with get, set" data.Name data.FullName data.Name
         w.printfn ""
         w.printfn "    static member Create%s () : %s =" data.Name data.FullName
```

Comment:
If I understand correctly, you might want to extend this method instead.
It's the one with the upcast.

Reply:
Yeah, I went with the public setter. I first tried adding a static extension method of the same name in the same namespace hoping that would override the default static methods in ViewBuilders(), but that didn't work.